src/Workleap.OpenApi.MSBuild/Spectral/SpectralRunner.cs

```diff
@@ -66,10 +68,35 @@ public async Task RunSpectralAsync(IReadOnlyCollection<string> openApiDocumentPa
             CiReportRenderer.AttachReportToBuild(spectralReportPath);
             this._loggerWrapper.LogMessage("\n ****************************************************************", MessageImportance.High);
         }
-        
+
         this._diffCalculator.SaveCurrentExecutionChecksum(spectralRulesetPath, openApiDocumentPaths);
     }
-    
+
+    // Display previous spectral report and log warning if there are any previous spectral errors or warnings
+    private void DisplayPreviousSpectralReport(string documentPath)
+    {
+        var pattern = @"\d+ problem(|s) \((\d+) error(|s), (\d+) warning(|s), \d+ info(|s), \d+ hint(|s)\)";
+        var regex = new Regex(pattern);
+        var lines = File.ReadLines(this.GetReportPath(documentPath));
+        foreach (var line in lines)
+        {
+            var match = regex.Match(line);
+            if (match.Success)
+            {
+                var errors = int.Parse(match.Groups[2].Value);
+                var warnings = int.Parse(match.Groups[4].Value);
+                if (errors > 0 || warnings > 0)
+                {
+                    this._loggerWrapper.LogWarning(line);
```

Comment:
nit: can we prefix this warning with "Spectral" or "OpenAPI validation" (or else) that provides more context about this warning?

![image](https://github.com/user-attachments/assets/52034470-726d-466f-8ddb-5a2352aa5c12)

I know the report content is right above it, but it could be easier to understand at first what the warning is about, before starting to scroll up to understand what's happening.

Reply:
Yup: 
![image](https://github.com/user-attachments/assets/aadc90c4-7495-41ca-975e-1233b8840997)